app/views/users/_form.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= form_for @user do |f| %>
+  <div class="form-outline mt-4 mb-4">
+    <%= f.label t(".name") %>
+    <%= f.text_field :name, class: "form-control #{"is-invalid" if @user.errors.include?(:name)}" %>
+    <p class="error"><%= show_errors(@user, :name) %></p>
+  </div>
+  <div class="form-outline mb-4">
+    <%= f.label t(".password") %>
+    <%= f.password_field :password, class: "form-control #{"is-invalid" if @user.errors.include?(:password)}" %>
```

Comment:
"is-invalid" if @user.errors.include?(:password) cũng viết helper nhé
truyền vào là object với tên field

Reply:
để k bị lặp code

Reply:
error_class_field @user, :password

Reply:
@user.errors.include?(:password) em viết đoạn này thôi ạ c